schemas/org.gnome.shell.extensions.gtile.gschema.xml

```diff
@@ -38,133 +38,173 @@
 			<summary>Resize window - decrease height.</summary>
 		</key>
 		<key type="as" name="set-tiling">
-		  <default><![CDATA[['Return']]]></default>
+            <default><![CDATA[['Return']]]></default>
 			<summary>Tile window according to selection.</summary>
 		</key>
 		<key type="as" name="cancel-tiling">
 			<default><![CDATA[['Escape']]]></default>
 			<summary>Cancel tiling.</summary>
 		</key>
 		<key type="as" name="change-grid-size">
-		        <default><![CDATA[['Space']]]></default>
-		  	<summary>Loop through grid sizes.</summary>
+			<default><![CDATA[['Space']]]></default>
+			<summary>Loop through grid sizes.</summary>
 		</key>
 
 		<key type="as" name="preset-resize-1">
-		  <default><![CDATA[['<Super><Alt>KP_1']]]></default>
+            <default><![CDATA[['<Super><Alt>KP_1']]]></default>
 		  	<summary>Preset resize 1.</summary>
 		</key>
 		<key type="as" name="preset-resize-2">
-		        <default><![CDATA[['<Super><Alt>KP_2']]]></default>
+            <default><![CDATA[['<Super><Alt>KP_2']]]></default>
```

Comment:
Since this file uses tabs, whitespace should be kept consistent. All of the spaces above this line should be replaced with tabs and indenting should match.